src/app/static/src/app/components/adr/SelectDataset.vue

```diff
@@ -34,7 +34,7 @@
             <h4 v-if="!loading" v-translate="'browseADR'"></h4>
             <p v-if="loading" v-translate="'importingFiles'"></p>
             <div v-if="!loading">
-                <label for="datasetSelector" class="font-weight-bold" v-translate="'datasets'"></label>
+                <label class="font-weight-bold" v-translate="'datasets'"></label>
```

Comment:
This was raising an "improvement issue" in chrome dev console as it didn't match up with any input field

Reply:
I'm struggling with this one a bit, I can't work out how to get the ID onto the actual input field. And having a look through docs here https://vue-treeselect.js.org/ I'm struggling to find anything that might achieve that? Any bright ideas?

As it is with the `for` set that still wouldn't help with accessibility right as it isn't pointing to any input?

Reply:
unfortunately, digging through the source code theres no way to pass id onto the input tag https://github.com/megafetis/vue3-treeselect/blob/f577e47c448b3b9c162322c910b97a4a2ce027ef/src/components/SingleValue.vue#L30